server/src/main/java/com/sauron/model/entities/Bank.java

```diff
@@ -0,0 +1,38 @@
+package com.sauron.model.entities;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "BANKS")
+public class Bank {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id", unique = true, updatable = false, nullable = false)
+    private Long id;
+    private String name;
+    private String apiUrl;
```

Comment:
I think for this task instead of apiUrl bank should have loginUrl/consentScreenUrl. Login url is required as user has to somehow consent proper permissions and it is more specific than apiUrl. Persisting apiUrls can be a new topic for next meeting.

Reply:
I can change it, however it's still up to discussions so I don't want to put too much effort into it for now.